packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -187,7 +187,8 @@ export class FlexObjectNodeSchema<
 		// Stricter typing caused Specification to no longer be covariant, so has been removed.
 		public readonly objectNodeFields: ReadonlyMap<FieldKey, FlexFieldSchema>,
 	) {
-		super(builder, name, info, new ObjectNodeStoredSchema(objectNodeFields));
+		const fields = [...objectNodeFields].map(([k, v]) => [k, v.stored] as const);
```

Comment:
Since `Map.entries()` produces an iterable and the `Map()` constructor accepts an iterable, you could use our `mapIterable` utility here. Not that it really matters; it may (or may not) look nicer stylistically.

Reply:
I did a little benahcmark, and it looks like the perf is similar:

```typescript
	const testMap: Map<number, number> = new Map();
	for (let index = 0; index < 1000; index++) {
		testMap.set(index, index * 2);
	}

	benchmark({
		title: `mapIterable-spread`,
		benchmarkFn: () => {
			const m = new Map([...testMap].map(([k, v]) => [k, v] as const));
		},
	});

	benchmark({
		title: `mapIterable`,
		benchmarkFn: () => {
			const m = new Map(mapIterable(testMap, ([k, v]) => [k, v] as const));
		},
	});
```

> status  name                period (ns/op)  relative margin of error  iterations per batch  batch count  total time (s)
> ------  ------------------  --------------  ------------------------  --------------------  -----------  --------------
>     ✔   mapIterable-spread       37,468.38                    ±1.52%                 2,048           64            5.02
>     ✔   mapIterable              41,239.38                    ±0.97%                 2,048           30            2.64

I expected the generator in mapIterable to have much worse overhead than that, but I guess either other things are slowing both down or generators are faster than they used top be. hus I have switched it to mapIterable.

Reply:
Nice benchmark! That's good to know going forwards.